Dockerfile.build

```diff
@@ -1,20 +0,0 @@
-FROM maven:3.3.3-jdk-8
-
-ENV GAIA_HOME=/usr/local/gaia/
-
-RUN mkdir -p $GAIA_HOME
-WORKDIR $GAIA_HOME
-
-# speedup maven build, read https://keyholesoftware.com/2015/01/05/caching-for-maven-docker-builds/
-
-# selectively add the POM file
-ADD pom.xml $GAIA_HOME
```

Comment:
Good strategy, although it not enough since CicleCI does not cache docker images